model/prelude_mem.sail

```diff
@@ -60,13 +60,13 @@ struct RISCV_strong_access = {
    with additional information, provided we can supply a function that
    converts it into a bitvector.  Since we are just using xlenbits as a
```

Comment:
Need to update this comment.

Reply:
I have modified it, I don't know if this comment is appropriate